Traceback (most recent call last):
  File "snippet.py", line 41, in <module>
    stdout=subprocess.PIPE)
  File "/usr/local/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/local/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
